src/plugins/loan-channel-query-map.js

```diff
@@ -224,119 +247,108 @@ export default {
 					id: 63,
 					url: 'loans-for-clean-energy',
 					queryParams: 'status=fundRaising&riskRating=0,5&theme=Clean Energy',
-					algoliaParams: 'attributes=Clean%20Energy'
+					algoliaParams: 'attributes=Clean%20Energy',
+					flssLoanSearch: { theme: ['CLEAN ENERGY'] },
 				},
 				{
 					id: 66,
 					url: 'lenders-like-you-supported-these-loans',
 					queryParams: 'status=fundRaising&riskRating=0,5',
-					algoliaParams: ''
+					algoliaParams: '',
+					flssLoanSearch: {},
```

Comment:
This is a "reccommened" row that would come out of the ML service instead of FLSS, so it's not eligible either.

Reply:
Removed.